src/libtiled/mapreader.cpp

```diff
@@ -53,6 +53,9 @@ using namespace Tiled::Internal;
 namespace Tiled {
 namespace Internal {
 
+	QString mTilesetSourceFiles[MAX_TILESET_SOURCE_FILES];
+	int mNumTilesetSourceFiles;
```

Comment:
I use the `m` prefix for member variables, but you have just made these ones globals. Were they meant to be member variables?

Also, you could probably just merge these two variables into a single `QStringList`.